### PR TITLE
Update sidebar UI and Logo

### LINKED ISSUE #720: 🚀 Feature: Sidebar & Logo update
### 🔖 Feature description

 UI update for DocsGPT sidebar
 https://www.figma.com/file/OXLtrl1EAy885to6S69554/DocsGPT?type=design&node-id=0%3A1&mode=design&t=M8rjHZLwUfyo6UXk-1
 
 1. New sidebar for light and dark themes. 
 2. Replaced logo on "home" and "about" screens and also next to chat messages that are from DocsGPT
 
image
image

image
image

image
image

### 🎤 Why is this feature needed ?

UI/UX improvements

### ✌️ How do you aim to achieve this?

UI/UX improvements

### 🔄️ Additional Information

_No response_

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### Are you willing to submit PR?

None

### PR DESCRIPTION
**Pull request by lakshmi930**
Feature: Updating the Sidebar UI and logo

Will partially close #720

### CODE REVIEW

File: frontend/src/conversation/ConversationBubble.tsx
Comment (dartpain) [inline frontend/src/conversation/ConversationBubble.tsx:1]: No need to make changes here

**Review by dartpain — CHANGES_REQUESTED** [review]
Everything is good, just one minor change and ready to merge. Thank you!

File: frontend/src/conversation/ConversationBubble.tsx
Comment (lakshmi930) [reply frontend/src/conversation/ConversationBubble.tsx:1]: This was mentioned as part of the requirement in the issue to update the logo in the reply sent from DocsGPT:
image

Comment (dartpain) [reply frontend/src/conversation/ConversationBubble.tsx:1]: My bad, the dino looks inverted

Comment (dartpain) [reply frontend/src/conversation/ConversationBubble.tsx:1]: Screenshot 2023-10-30 at 09 23 26

Comment (lakshmi930) [reply frontend/src/conversation/ConversationBubble.tsx:1]: Okay, let me check this.

Comment (lakshmi930) [reply frontend/src/conversation/ConversationBubble.tsx:1]: Updated @dartpain , can you please check again? Also, how to test the dark mode? I see the designs ready but no way to switch the mode on the ui.

Comment (dartpain) [reply frontend/src/conversation/ConversationBubble.tsx:1]: No dark mode, but its ok, ill link backend

**Review by dartpain — APPROVED** [review]

**Comment by dartpain** [comment]
Thank you so much!

**Comment by lakshmi930** [comment]
Thank you for the quick review and release too @dartpain. This was a great chance to work on Tailwind for me.

**Pull request merged**